src/app/cli/src/app.rs

```diff
@@ -586,6 +588,8 @@ pub fn configure_server_catalog(base_catalog: &Catalog) -> CatalogBuilder {
 
     kamu_webhooks_services::register_dependencies(&mut b);
 
+    kamu_adapter_auth_web3::register_dependencies(&mut b);
```

Comment:
I thought it would be logical to initialize auth adapters (GitHub, Web3) somewhere nearby.

Reply:
Fair comment -- Moved

Reply:
UPD: reverted -- `Web3WalletAuthenticationProvider` depends on `ServerUrlConfig` (a server component)